### PR TITLE
remove simple form from documentation and update screenshots (+some changes in news intro)

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1155,58 +1155,34 @@ the link.
...
-From a space's acivity stream, you are able to share a news with the space’s members. 
-Publishing a news allows you to easily write, broadcast, pin and share communication content.
-
-
-Posting a simple news from short form
---------------------------------------
 
-To post an article with other space’s members, click on News tab from the activity composer, the tab contains :
+It is possible to write an article directly from a space composer. This feature is intended to improve and simplify the way information is shared. It allows contributors to create content without having to be CMS experts. It offers an inclusive and collaborative way to manage communication.
+Publishing a news allows you to easily write, broadcast  and share communication content into other spaces. You can also make it visible to all platform users by pinning articles into home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Publishing a news lets you easily write, broadcast  and share communication content into other spaces. You can also make it visible to all platform users by pinning articles on the home page.
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1155,58 +1155,34 @@ the link.
...
-	
-From a space's acivity stream, you are able to share a news with the space’s members. 
-Publishing a news allows you to easily write, broadcast, pin and share communication content.
-
-
-Posting a simple news from short form
---------------------------------------
 
-To post an article with other space’s members, click on News tab from the activity composer, the tab contains :
+It is possible to write an article directly from a space composer. This feature is intended to improve and simplify the way information is shared. It allows contributors to create content without having to be CMS experts. It offers an inclusive and collaborative way to manage communication.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
It is possible to write an article directly from a space composer. This feature is intended to improve and simplify the way information is shared. Users can contribute content intuitively without any former training to the content management system. News offers an inclusive and collaborative way to manage communication.
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1155,58 +1155,34 @@ the link.
...
+Publishing a news allows you to easily write, broadcast  and share communication content into other spaces. You can also make it visible to all platform users by pinning articles into home page.
 
+To post an article, click "write an article" located at the buttom of the space's composer.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
To post a news article, click "write an article" located at the bottom of the space's composer. 
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1155,58 +1155,34 @@ the link.
...
+|image156|
 
-- A **Content** field : Allows you to enter the content of the News. No limitation for the number of characters.
+The creation form is opened in a new tab.
 
-- A **Pin article** Check box.
+|image159|
 
-- A **More icon** with a tooltip “more options”: Permits to open the creation full form.
+Write or paste in your article. Add a title, summary, image, and content as desired, then press the "Post" button in the top-right corner of the screen.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Write or paste your article content in. Choose a title, a summary and an illustrative image as desired, then press the "Post" button in the top-right corner of the screen. 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
first pass of proofreading and language suggestions

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -2221,8 +2192,6 @@ to display the original timestamp of the comment.
 .. |image170| image:: images/sharenews/pin_activity.png
 .. |image171| image:: images/sharenews/success_pin.png
 .. |image172| image:: images/sharenews/bloc_news.png
-.. |image173| image:: images/sharenews/saving_shortform.png
-.. |image174| image:: images/sharenews/saved_shortform.png
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst:2225]: These images files must be removed.

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 5 above)
Comment (samehaidi) [reply docs/GettingStarted.rst:2225]: done

**Review by plamarque — APPROVED** [review]
LGTM

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**